flink-formats/flink-orc/src/test/java/org/apache/flink/orc/OrcFileSystemITCase.java

```diff
@@ -69,22 +72,18 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 /** ITCase for {@link OrcFileFormatFactory}. */
-@RunWith(Parameterized.class)
+@ExtendWith(ParameterizedTestExtension.class)
 public class OrcFileSystemITCase extends BatchFileSystemITCaseBase {
 
-    @ClassRule public static final TemporaryFolder TEMPORARY_FOLDER = new TemporaryFolder();
+    @TempDir public static java.nio.file.Path temporaryFolder;
 
-    private final boolean configure;
+    @Parameter public boolean configure;
 
-    @Parameterized.Parameters(name = "{0}")
+    @Parameters(name = "{0}")
```

Comment:
```suggestion
    @Parameters(name = "configure={0}")
```

Reply:
Somehow I forgot to synchronize the changes here 😞